src/helpers/checkSlug.ts

```diff
@@ -0,0 +1,6 @@
+export function checkSlug(slug: string | undefined): boolean {
+  if (slug !== '' && !slug?.match(/\//)) {
```

Comment:
This line was extracted from where it existed previously:

https://github.com/codecov/uploader/blob/12dccfc8752f374428180d8cbde95887e7374c6c/src/index.ts#L328

Reply:
No action is required for this annotation.